pkg/cmd/completion/cmd.go

```diff
@@ -28,6 +28,7 @@ func NewCmd() *cobra.Command {
 		Example:      eg,
 		SilenceUsage: true,
 		ValidArgs:    []string{"bash", "zsh"},
+		Args:         cobra.OnlyValidArgs,
```

Comment:
This finally results in proper error when calling `ike completion fish` for example.